desk/app/groups.hoon

```diff
@@ -3527,13 +3520,17 @@
       =.  go-core  (go-response %channel nest [%del-readers roles.u-channel])
       ?:  go-our-host  go-core
       ::
+      ?.  (has:by-ch nest)
+        (go-restart-updates 'missing channel')
```

Comment:
Should be `'missing channel deleted readers'` or similar to distinguish this branch from others.